.github/workflows/build-test.yaml

```diff
@@ -287,18 +287,17 @@ jobs:
           export BUILD_LOG_NAME=${{ inputs.mode }}-${{ inputs.target }}-${{ inputs.gcchash }}-${{ inputs.multilib }}-build-log
           echo "build_log_name=$BUILD_LOG_NAME" >> $GITHUB_OUTPUT
 
-      # run-frequent uses rv64gcv, run-weekly uses rv64gcv_zvl
+      # run-frequent uses rv64gcv, run-weekly uses target
```

Comment:
This is in the `run-on-self-hosted` job. The only self-hosted jobs are for vector targets